timelock-api/src/main/conjure/timelock-management-api.yml

```diff
@@ -23,7 +23,8 @@ services:
         http: POST /getActiveNamespaces
         returns: set<string>
         docs: |
-          The endpoint loads all namespaces with new timestamps since the last restart. 
+          The endpoint loads all namespaces that have requested new timestamps within the past 6 hours
```

Comment:
It's any interaction with the namespace (barring maybe one or two uncommon ones), not just getting a new timestamp!